remixer_example/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I'll add that when we are properly supporting this, right now I'm adding the backbone, but it's also useless to the user without the remote controller. 

When it's ready (and public) I will add these instructions to the readme (or start a Wiki since options start branching out).